container_linux.go

```diff
@@ -135,6 +135,7 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 	cmd.Stdin = p.Stdin
 	cmd.Stdout = p.Stdout
 	cmd.Stderr = p.Stderr
+	cmd.ExtraFiles = p.ExtraFiles
```

Comment:
A nil check is needed i think:

```
 extraFiles = []*os.File{childPipe}
 if p.ExtraFiles != nil {
         append(extraFiles, p.ExtraFiles)
 }
 cmd.ExtraFiles = extraFiles
```
but I need to figure out, if the passed files gets the right inode numbers (from 3 to n)